From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Paul Philippov <paul@themactep.com>
Date: Mon, 17 Mar 2025 17:53:20 +0000
Subject: [PATCH] Fix SHA1 header include for tools build

The lib/sha1.c file was using #include "sha1.h" which looks for the header
in the local directory first. During the tools build phase, this causes
compilation errors because the header is located in include/sha1.h.

This patch changes the include to use angle brackets so the compiler looks
in the system include directories where the header is properly available.

Fixes compilation error:
error: unknown type name 'sha1_context'

Signed-off-by: Thingino Project <dev@thingino.com>
---
 lib/sha1.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/sha1.c b/lib/sha1.c
index 1234567..abcdefg 100644
--- a/lib/sha1.c
+++ b/lib/sha1.c
@@ -36,7 +36,7 @@
 #include <string.h>
 #endif /* USE_HOSTCC */
 #include <watchdog.h>
-#include "sha1.h"
+#include <sha1.h>

 /*
  * 32-bit integer manipulation macros (big endian)
